core/src/style/polylineStyle.cpp

```diff
@@ -205,8 +205,9 @@ void PolylineStyle::buildLine(const Line& _line, const DrawRule& _rule, const Pr
         if (evalStyleParamWidth(StyleParamKey::outline_width, _rule, _tile, widthOutline, dWdZOutline) &&
             ((widthOutline > 0.0f || dWdZOutline > 0.0f)) ) {
 
-            widthOutline += width;
-            dWdZOutline += dWdZ;
+            // Note: this must update width and dWdZ as they are captured (and used) by builder
+            width += widthOutline;
```

Comment:
should `width` and `dWdZ` be zeroed-out before this?

And this is only when `Builder::buildPolyLine` is again called for re-triangulation.